assets/chat/js/chat.js

```diff
@@ -635,6 +635,13 @@ class Chat {
       .catch(() => {});
   }
 
+  setPreLoginText() {
+    const chatText = ChatStore.read('chat.preLoginText') || '';
+    this.input.val(chatText);
+    ChatStore.remove('chat.preLoginText');
+    this.input.focus();
```

Comment:
The chat input autofocuses on desktop, so focusing it manually isn't necessary.

And although I didn't test it, this might also focus the input/bring up the virtual keyboard on mobile when the page loads, which is undesirable for UX reasons.

Reply:
addressed